src/inspector/view/qml/MuseScore/Inspector/general/playback/PlaybackGeneralSettings.qml

```diff
@@ -47,62 +47,115 @@ Item {
 
         spacing: 12
 
+        property var fullModel: [
+            { typeRole: Inspector.TYPE_NOTE, componentRole: noteSection },
+            { typeRole: Inspector.TYPE_ARPEGGIO, componentRole: arpeggioSection },
+            { typeRole: Inspector.TYPE_FERMATA, componentRole: fermataSection },
+            { typeRole: Inspector.TYPE_BREATH, componentRole: pausesSection },
+            { typeRole: Inspector.TYPE_GLISSANDO, componentRole: glissandoSection },
+            { typeRole: Inspector.TYPE_GRADUAL_TEMPO_CHANGE, componentRole: tempoChangeSection }
+        ]
+
+        function getVisibleModel() {
+            var visibleModel = [];
+            var currType = null;
+            for (let i = 0; i < fullModel.length; i++) {
+                currType = root.proxyModel ? root.proxyModel.modelByType(fullModel[i].typeRole) : null;
+                if (Boolean(currType) && !currType.isEmpty) {
+                    visibleModel.push(fullModel[i]);
+                }
+            }
+            return visibleModel;
+        }
+
+        Repeater {
+            id: repeater
+
+            model: contentColumn.getVisibleModel()
+
+            delegate: Column {
+                id: itemColumn
+
+                width: parent.width
+                height: childrenRect.height
+
+                spacing: contentColumn.spacing
+
+                visible: expandableLoader.active
+
+                SeparatorLine {
+                    anchors.margins: -12
+                    visible: model.index>0
+                }
+
+                Loader {
+                    id: expandableLoader
+
+                    property var itemModel: root.proxyModel ?  proxyModel.modelByType(modelData.typeRole) : null
+
+                    width: parent.width
+
+                    sourceComponent: modelData["componentRole"]
+
+                    onLoaded: {
+                        expandableLoader.item.model = expandableLoader.itemModel
```

Comment:
Now we can add more stuff here (and remove it from components):

 expandableLoader.item.navigation.panel = root.navigationPanel
  expandableLoader.item.navigation.row = root.navigationRowStart + model.index * 1000